Move plugin URL to github (#358)
jenkinsci/pipeline-model-definition-plugin: README.md

# Declarative Jenkins Pipelines

> [![Jenkins Plugin](https://img.shields.io/jenkins/plugin/v/pipeline-modeldefinition)](https://plugins.jenkins.io/pipeline-modeldefinition-plugin)

Reviewer: ```suggestion
[![Jenkins Plugin](https://img.shields.io/jenkins/plugin/v/pipeline-model-definition)](https://plugins.jenkins.io/pipeline-model-definition)
```

> [![Changelog](https://img.shields.io/github/v/tag/jenkinsci/pipeline-modeldefinition-plugin?label=changelog)](https://github.com/jenkinsci/pipeline-modeldefinition-plugin/blob/master/CHANGELOG.md)

Reviewer: ```suggestion
[![Changelog](https://img.shields.io/github/v/tag/jenkinsci/pipeline-model-definition-plugin?label=changelog)](https://github.com/jenkinsci/pipeline-model-definition-plugin/blob/master/CHANGELOG.md)
```

> [![Jenkins Plugin Installs](https://img.shields.io/jenkins/plugin/i/pipeline-modeldefinition?color=blue)](https://plugins.jenkins.io/pipeline-modeldefinition)

Reviewer: ```suggestion
[![Jenkins Plugin Installs](https://img.shields.io/jenkins/plugin/i/pipeline-model-definition?color=blue)](https://plugins.jenkins.io/pipeline-model-definition)
```

A config-like syntax for defining Pipelines. See [the docs on jenkins.io](https://jenkins.io/doc/) for more information on how to use it.

This plugin requires Jenkins 2.121.1 or later

**WARNING**:
Older versions of this plugin may not be safe to use. Please review the
following warnings before using an older version:

-   [Script Security sandbox
    bypass](https://jenkins.io/security/advisory/2019-01-08/#SECURITY-1266)

## Documentation

> * [Changelog](https://github.com/jenkinsci/pipeline-modeldefinition-plugin/blob/master/CHANGELOG.md)

Reviewer: ```suggestion
* [Changelog](https://github.com/jenkinsci/pipeline-model-definition-plugin/blob/master/CHANGELOG.md)
```

> * [Developer Guide](https://github.com/jenkinsci/pipeline-modeldefinition-plugin/blob/master/DEV_GUIDE.md)

Reviewer: ```suggestion
* [Developer Guide](https://github.com/jenkinsci/pipeline-model-definition-plugin/blob/master/DEV_GUIDE.md)
```

> * [Instructions for extending this plugin](https://github.com/jenkinsci/pipeline-modeldefinition-plugin/blob/master/EXTENDING.md)

Reviewer: ```suggestion
* [Instructions for extending this plugin](https://github.com/jenkinsci/pipeline-model-definition-plugin/blob/master/EXTENDING.md)
```
